### PR TITLE
Upgrade OS version to bionic for Docker image

### PR DESCRIPTION
**Pull request by kchygoe**
## What this patch does to fix the issue.
- Upgrade Base OS version from xenial to bionic(Ubuntu 18.04)
- Drop python2 support from device side
- Add uninstallation of enum34 task in CI because this package makes other packages unavailable to be installed
- font package is required because bionic image doesn't provide it by default

## Link to any relevant issues or pull requests.
Related to #1138

### CODE REVIEW

**Comment by kchygoe** [comment]
@iizukak Sorry, could you tell me why g++-9 is now installed in Docker image?
https://github.com/blue-oil/blueoil/pull/690

**Comment by kchygoe** [comment]
need to uninstall enum34 on device to install newer pip packages

**Comment by kchygoe** [comment]
ya, I'm thinking about that

**Comment by kchygoe** [comment]
- Regarding gcc version, I will create another PR.
- requirements.txt will be updated in another PR, too

File: .buildkite/blueoil-pipeline.yml [hunk 1]
```diff
@@ -68,12 +68,15 @@ steps:
       python3 -m venv blueoil_inference
       source blueoil_inference/bin/activate
       pip3 install -U pip
+      pip3 uninstall -y enum34
```
Comment (hadusam) [inline .buildkite/blueoil-pipeline.yml:71]: I am not sure why we need this line.
Do we need this line at this time?

Comment (hadusam) [reply .buildkite/blueoil-pipeline.yml:71]: If this uninstall operation is temporary for miss much of requirements.txt between each CI of existing PRs, I think it is better to add a comment or create an issue to remove this line 👍

Comment (kchygoe) [reply .buildkite/blueoil-pipeline.yml:71]: thx, I will make an issue

Comment (kchygoe) [reply .buildkite/blueoil-pipeline.yml:71]: https://github.com/blue-oil/blueoil/issues/1143

**Pull request merged**